cltk/prosody/old_norse/verse.py

```diff
@@ -0,0 +1,214 @@
+"""Verse structures of Old Norse poetry"""
+
+from math import floor
+from cltk.phonology.old_norse.transcription import Transcriber,  old_norse_rules, IPA_class, DIPHTHONGS_IPA_class, \
+    DIPHTHONGS_IPA
+from cltk.phonology.syllabify import Syllabifier
+from cltk.tokenize.word import tokenize_old_norse_words
+from cltk.utils.cltk_logger import logger
+
+__author__ = ["Clément Besnier <clemsciences@aol.com>", ]
+
+
+class VerseManager:
+    """
+    * Fornyrðislag
+    * Ljóðaháttr
+    """
+    @staticmethod
+    def is_fornyrdhislag(text: str):
```

Comment:
From what I get, you judge whether the meter is fornyrdhislag, solely upon the number of lines. While this could do, this still requires properly punctuated text, as this will fail for any text without newlines. This may be a overkill, but I think it would be worth to try and see methods for probabilistic metric partition, or at least add supplementary checks (check for alliteration rules?).

Reply:
Fair enough, I guess it would make more sense to keep things as is and worry about generalizing this later. But yes, I would too make sure to mention the formatting restrictions in the docs